java/java.lsp.server/nbcode/integration/src/org/netbeans/modules/nbcode/integration/LspAnalyserConfig.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.netbeans.modules.nbcode.integration;
+
+import org.netbeans.modules.java.source.usages.BinaryAnalyser;
+import org.openide.util.lookup.ServiceProvider;
+
+/**
+ *
+ * @author Dusan Balek
+ */
+@ServiceProvider(service = BinaryAnalyser.Config.class, position = 1000)
+public class LspAnalyserConfig extends BinaryAnalyser.Config {
```

Comment:
Other `LspXYZ` services are provided as `public abstract` by `java.lsp.server` and then subclassed in the `nbcode` and registered with the `@ServiceProvider` - exactly to avoid exposing internal details (lsp4j, etc.) to `nbcode` suite.

Reply:
Done.